specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip:checkNumberRouting": {
+      "get": {
+        "tags": [
+          "NumberRoutingValidation"
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckNumberRouting",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
```

Comment:
This doesn't seem to be a path parameter?

Reply:
That's right - it used to be before @prprabhu-ms 's review (namely `/sip/checkRouting/{targetNumber}`) but as Prathmesh said it shouldn't be path since it's not a resource. Just fixed this now